src/platform-apps/channels/channel-view.tsx

```diff
@@ -104,6 +106,15 @@ export class ChannelView extends React.Component<Properties, State> {
     );
   }
 
+  renderChatWindow() {
+    return (
+      <ChatWindow
+        isUserConnected={true}
+        onSubmit={this.props.sendMessage}
+      />
```

Comment:
i'm not sure this is the right abstraction here. i think that ChannelView and ChannelWindow are actually the same thing. i think we can probably just pull the input into the channel view, instead of needing to also add the ChatWindow component?